engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -355,34 +359,36 @@ case class WorkflowExecutionActor(workflowDescriptor: EngineWorkflowDescriptor,
     if (runnableCalls.nonEmpty) workflowLogger.info("Starting calls: " + runnableCalls.mkString(", "))
 
     // Each process returns a Try[WorkflowExecutionDiff], which, upon success, contains potential changes to be made to the execution store.
-    val executionDiffs = runnableScopes map {
-      case k: CallKey if isInBypassedScope(k, data) => processBypassedScope(k, data)
-      case k: DeclarationKey if isInBypassedScope(k, data) => processBypassedScope(k, data)
-      case k: BackendJobDescriptorKey => processRunnableJob(k, data)
-      case k: ScatterKey => processRunnableScatter(k, data, isInBypassedScope(k, data))
-      case k: ConditionalKey => processRunnableConditional(k, data)
-      case k: CollectorKey => processRunnableCollector(k, data, isInBypassedScope(k, data))
-      case k: SubWorkflowKey => processRunnableSubWorkflow(k, data)
-      case k: StaticDeclarationKey => processRunnableStaticDeclaration(k)
-      case k: DynamicDeclarationKey => processRunnableDynamicDeclaration(k, data)
-      case k =>
-        val exception = new UnsupportedOperationException(s"Unknown entry in execution store: ${k.tag}")
-        self ! JobInitializationFailed(k, exception)
-        Failure(exception)
+    val diffs = runnableScopes map { scope =>
+      scope -> Try(scope match {
+        case k: CallKey if isInBypassedScope(k, data) => processBypassedScope(k, data)
+        case k: DeclarationKey if isInBypassedScope(k, data) => processBypassedScope(k, data)
+        case k: BackendJobDescriptorKey => processRunnableJob(k, data)
+        case k: ScatterKey => processRunnableScatter(k, data, isInBypassedScope(k, data))
+        case k: ConditionalKey => processRunnableConditional(k, data)
+        case k: CollectorKey => processRunnableCollector(k, data, isInBypassedScope(k, data))
+        case k: SubWorkflowKey => processRunnableSubWorkflow(k, data)
+        case k: StaticDeclarationKey => processRunnableStaticDeclaration(k)
+        case k: DynamicDeclarationKey => processRunnableDynamicDeclaration(k, data)
+        case k => Failure(new UnsupportedOperationException(s"Unknown entry in execution store: ${k.tag}"))
+      }).flatten
+    } map {
+      case (_, Success(value)) => Success(value)
+      case (scope, Failure(throwable)) =>
+        self ! JobInitializationFailed(scope, throwable)
+        Failure(throwable)
+    } collect {
+      case Success(value) => value
```

Comment:
A couple things:

- The other jobs have _already_ started by the point we do a `Try.sequence`. It's too late to stop them from launching. The EJEAs are already off to the races.
- Even when we do detect that something went wrong in "1 of 2000" jobs that just started, what is the duty of _this_ method?

Before, the method was throwing an exception killing the whole actor. The workflow wasn't actually stopped, and was left in an infinite "running" state. But yes even-more jobs didn't start. 😉 

Elsewhere in this WEA is other code (that isn't 100% [imho](https://github.com/broadinstitute/cromwell/issues/2029)) that decides whether to fail-fast or fail-slow when a single job fails. Again imho it's _that_ code that should decide what to do about the individual job failures, which is why I chose to send out a message to self.

So, happy to tech talk this, but I currently believe this method should be messaging, and not throwing exceptions.